stories/object-list/object-list.stories.js

```diff
@@ -0,0 +1,1194 @@
+          </div>
+        </div>
+        <div class="fd-object-list__row">
+          <div class="fd-object-list__row-left">
+            <div class="fd-object-list__object-attribute">
+              Second Attribute
+            </div>
+          </div>
+          <div class="fd-object-list__row-right">
+            <span class="fd-object-status fd-object-status--critical">
+              <span class="fd-object-status__text">Critical</span>
+            </span>
+          </div>
+        </div>
+        <div class="fd-object-list__row">
+          <div class="fd-object-list__row-left">
+            <div class="fd-object-list__object-attribute">
+              Third Attribute
+            </div>
+        </div>
+      </div>
+    </div>
+    </div>
+  </li>
+  <li role="listitem" tabindex="0" class="fd-list__item fd-object-list__item">
+    <div class="fd-object-list__container">
+      <div class="fd-object-list__intro">
+       Optional inline text very very very long
+      </div>
+      <div class="fd-object-list__header">
+        <span class="fd-avatar fd-avatar--s"
+         style="background-image: url('http://lorempixel.com/400/400/nature/4/')">
+        </span>
+        <div class="fd-object-list__header-left">
+         <div class="fd-object-identifier fd-object-list__object-identifier">
+          <p class="fd-object-identifier__title">
+           Fitbit Versa Smart Watch, Black Fitbit Versa Smart Watch
+          </p>
+         </div>
+        </div>
+        <div class="fd-object-list__header-right">
+          <span class="fd-object-number fd-object-number--critical fd-object-list__object-number">
+            <span class="fd-object-number__text">457.00</span>
+            <span class="fd-object-number__unit">Euro</span>
+          </span>
+        </div>
+      </div>
+      <div class="fd-object-list__content">
+        <div class="fd-object-list__row">
+          <div class="fd-object-list__row-left">
+            <div class="fd-object-list__object-attribute">
+              
+              First Attribute
+            </div>
+          </div>
+          <div class="fd-object-list__row-right">
+            <div class="fd-object-marker">
+              <i class="fd-object-marker__icon sap-icon--user-edit" role="presentation"></i>
+                <span class="fd-object-marker__text">Draft</span>
+            </div>
+          </div>
+        </div>
+        <div class="fd-object-list__row">
+          <div class="fd-object-list__row-left">
+            <div class="fd-object-list__object-attribute">
+              Second Attribute
+            </div>
+          </div>
+        </div>
+        <div class="fd-object-list__row">
+          <div class="fd-object-list__row-left">
+            <div class="fd-object-list__object-attribute">
+              Third Attribute
+            </div>
+        </div>
+      </div>
+    </div>
+    </div>
+  </li>
+</ul>
+`;
+
+object.storyName = 'Object';
```

Comment:
**Standard**
The standard object list item displays a title, introductory text, an avatar and object display components. It is recommended to display avatar in size S, by adding the `fd-avatar--s` modifier class to the `fd-avatar` element.